### PR TITLE
Heb updates

### PR DESCRIPTION
**Pull request by yosilevy**
Various updates to have Heb include all the functionality as in English.

### CODE REVIEW

**Comment by yosilevy** [comment]
Please hold on. I need to review conflicts first (a little later today).

**Comment by yosilevy** [comment]
@leranp @haim-b I merged what you recently did keeping most of it over my stuff + combining/making more explicit. Thanks

**Comment by yosilevy** [comment]
I will finish tests tomorrow. Sorry.

File: sentences/he/_common.yaml [hunk 1]
```diff
@@ -46,60 +46,127 @@ lists:
         out: "celsius"
   brightness_level:
     values:
-      - in: ( מקסימלי | הכי גבוה | עד הסוף | מרבי | מירבי | למקסימום)
+      - in: (מקס | מקסימום | הגבוה | הגבוה ביותר | הגבוהה ביותר)
```
Comment (leranp) [inline sentences/he/_common.yaml:49]: Please keep the other words in the list

File: sentences/he/_common.yaml [hunk 2]
```diff
@@ -46,60 +46,127 @@ lists:
...
         out: 100
-      - in: ( מינימלי | הכי נמוך | למינימום)
+      - in: (מינ | מינימום | הנמוך | הנמוך ביותר | הנמוכה ביותר)
```
Comment (leranp) [inline sentences/he/_common.yaml:51]: Please keep the other words in the list

**Review by leranp — COMMENTED** [review]
Thank you for your work, you need to change the test files with your changes

File: sentences/he/_common.yaml (hunk 1 above)
Comment (yosilevy) [reply sentences/he/_common.yaml]: Done

File: sentences/he/_common.yaml (hunk 2 above)
Comment (yosilevy) [reply sentences/he/_common.yaml]: Done

**Comment by yosilevy** [comment]
@leranp I need help understanding why prettier is failing... thanks

**Comment by leranp** [comment]
> @leranp I need help understanding why prettier is failing... thanks

Please execute this command to check what needs to fix

Python3 tests --language he -k HassGetState

Iet me know if you need anything , i will be able to help only tomorrow

**Comment by yosilevy** [comment]
Thenks @leranp I get this:
```
/usr/local/bin/python3: can't find '__main__' module in '/home/yosi/dev/intents/.'
```
Probably something not setup fully. I'll see if there are docs. Otherwise if you can help that'd be great. Toda

**Comment by leranp** [comment]
> Thenks @leranp I get this:
> 
> ```
> /usr/local/bin/python3: can't find '__main__' module in '/home/yosi/dev/intents/.'
> ```
> 
> Probably something not setup fully. I'll see if there are docs. Otherwise if you can help that'd be great. Toda

it looks like your fork isn't sync with the main branch , please open a new fork and restore the files that was changed.

**Comment by yosilevy** [comment]
It doesn't work even with a fresh copy and without my changes. Please try running
```
python3 tests --language he -k HassGetState
```

**Comment by yosilevy** [comment]
@leranp solved. Now working on tests. There are weird things there like
``` תפתח את ה המתג ```

**Comment by yosilevy** [comment]
@leranp can you see if tests run for you please and point me in the right direction. I need to be able to run them locally.
Right now it is weird - when doing פתח את התריס in command line I get slot name = תריס but when the test runs it then it gets מתג.

**Comment by leranp** [comment]
> @leranp can you see if tests run for you please and point me in the right direction. I need to be able to run them locally.
> Right now it is weird - when doing פתח את התריס in command line I get slot name = תריס but when the test runs it then it gets מתג.
> 

I will try to simulate the code tomorrow .
Next time please add changes in separate PR, then you can handle the errors more efficiently :-)

**Comment by yosilevy** [comment]
Thanks. Will do next time. I wanted to do a full pass through and I didn't think it would get ugly...

**Comment by yosilevy** [comment]
@leranp I'd really appreciate help. Right now tests are failing and I'm not sure how to fix and I need to push to run each set of tests. If you could give me instructions on setup so that tests run locally it would be great. Thanks.

**Comment by leranp** [comment]
> @leranp I'd really appreciate help. Right now tests are failing and I'm not sure how to fix and I need to push to run each set of tests. If you could give me instructions on setup so that tests run locally it would be great. Thanks.

Let's continue the discussion on discord, i have sent you a message

**Comment by leranp** [comment]
please add a separate PR for each of the Intent , for example HassGetState.

**Pull request closed without merging**